src/test/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/segmentgroup/LabResultTest.java

```diff
@@ -68,10 +81,20 @@ void testGetLaboratoryInvestigationResult() {
             "RSL+NV+11.9:7++:::ng/mL+HI",
             "ignore me"
         ));
-        assertThat(labResult.getInvestigationResult())
-            .isPresent()
-            .map(LaboratoryInvestigationResult::getValue)
-            .contains("NV+11.9:7++:::ng/mL+HI");
+        var labResultInvestigationResult = assertThat(labResult.getInvestigationResult()).isPresent();
```

Comment:
I really like this approach of re-using an intermediate assertj object! Haven't seen it before 👍🏼